### DISCUSSION TITLE
Multiple Application on single client

### DISCUSSION THREAD

**Discussion by rickvian** [featurehub-io/featurehub-javascript-sdk#106] [General]
i have Next.js App that want to consume flags from 2 different projects in feature hub,

so the concept is 1 project is for FE flags, the other are shared flag across team member  (BE and android platform)

one is currently consumed using long polling, the shared one is not yet consumed, i was working on setting it up...

is there a reference of do and don'ts of applying this?

<img width="448" alt="FeatureHub" src="https://user-images.githubusercontent.com/8034141/208564018-b9c9b9af-a9bd-4815-b3b4-eb40991b2bf0.png">

**Comment by rvowles** [comment]
Hi @rickvian  - the repository is kept separate in all SDKs (although I'm not 100% sure on the Go one), so you can have multiple Edge connectors feeding data. The SSE connector (I think this is what you mean by long polling?) can only feed a single API key, but you can feed two (or more) connections into the same repository and they will just amalgamate. the GET api accepts multiple API keys and so does this for you automatically.
